src/test/java/org/jitsi/meet/test/ChatPanelTest.java

```diff
@@ -76,27 +76,21 @@ public void testChatPanel()
             "The chat panel should be initially closed");
 
         chatPanel.clickToolbarButton();
-        TestUtils.waitMillis(1000);
-        assertTrue(
-            chatPanel.isOpen(),
-            "The chat panel should be open after clicking the button");
+
+        // The chat panel should be open after clicking the button
+        WebDriverWait wait = new WebDriverWait(participant.getDriver(), 2);
+        wait.until((ExpectedCondition<Boolean>) d -> chatPanel.isOpen());
 
         chatPanel.pressShortcut();
```

Comment:
do you know why we first click the toolbar button and then use the shortcut ? wouldn't that result in open and close ?

Reply:
oh I see it's used to close it...